edk2toolext/nuget_publishing.py

```diff
@@ -217,6 +242,17 @@ def _MakeNuspecXml(self, ContentDir, ReleaseNotesText=None):
         meta.find("version").text = self.NewVersion
         meta.find("authors").text = self.ConfigData["author_string"]
         meta.find("projectUrl").text = self.ConfigData["project_url"]
+        repository_item_present = bool([k for k in self.ConfigData.keys() if "repository" in k.lower()])
+        if repository_item_present:
```

Comment:
Repository should be removed if no items are present. `meta.remove()`

Reply:
Good point. I'll add that.

Reply:
Done in PR commit https://github.com/tianocore/edk2-pytool-extensions/pull/429/commits/2682256cc76730f18db5526ac045b621f119f4db